### ISSUE TITLE
About the heda_tilt in manipulation part

### ISSUE THREAD

**Issue by yyy6812** [ok-robot/ok-robot#13]
Hi,
Thank you very much for such a good open source work. Due to my limited knowledge in this area, I have few questions when reading your code.

1. Do all the pose and point in the center_robot/pick/place output are relative to camera coordinate system?

2.  I wan to use the ok-robot-manipution in our robot controller project, but I have a question about the `head_tilt` parameter. 
As this parameter is not always included by the camera intrinsics, and the Anygrasp code didn't use it to generate a grasp pose originally.
Does there any formula to help understand how `heda_tilt ` is applicated in your work, and if our camera doesn't have the `heda_tilt` parameter, how could we to alter the manipulation part code ?

3. What does `crop'flag` mean and what is its purpose to use?

Thanks again!  Looking forward to your reply.

**Comment by yyy6812** [comment]
When using the example_data, you scaled the depths from the depth image and scaled the head_tilt using `head_tilt / 100`, but when using the communication, you left out the scale process，could you tell the difference between them please? Thank you very much!

**Comment by yas777** [comment]
Hi yu3jun,

1. Yes, The poses and points for all three functions are in camera coordinate frame.
2. head_tilt is the camera tilt [radians] in vertical direction. It is not exactly a camera intrinsic parameter, we get this information from camera state obtained from hello stretch apis and send it along with intrinsic parameters from robot side.

- **Re scaling** - Essentially the colors should be in 0-1 range, depth should be in meters and head_tilt in radians. We scaled the colors, depth and head_tilt as we were saving the data in different scales. 

3. It was not used in the final version but initially we attempted to crop out just the object point cloud and send it to anygrasp code [instead of sending the whole image point cloud] whenever no poses were found as a failure correction mechanism. The crop_flag is to show whether this mechanism is being applied or not. However, we found out that this is not improving the performance significantly. So we removed this flag in the final version of code.

let us know if you have more questions,
